spanner/metrics_test.go

```diff
@@ -148,9 +148,8 @@ func TestNewBuiltinMetricsTracerFactory(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.desc, func(t *testing.T) {
-			if test.setEmulator {
```

Comment:
can you use `runOnlyInEmulator` and set the value to `isEmulatorEnvSet()` so that you can avoid having two checks.